app/assets/javascripts/navbar.js

```diff
@@ -0,0 +1,13 @@
+$( document ).on('turbolinks:load', function() {

```

Comment:
@patri15 try to structure your code in a modular menu (break it into smaller functions) and call them in a `init` method. Also there are some good tips here on how to write javascript: https://github.com/ryanmcdermott/clean-code-javascript#functions